spek-junit-platform-engine/src/main/kotlin/org/jetbrains/spek/engine/ThrowableCollector.kt

```diff
@@ -25,10 +25,10 @@ class ThrowableCollector {
 
     fun isEmpty() = throwable == null
 
-
     fun assertEmpty() {
-        if (throwable != null) {
-            throw throwable!!
+        throwable?.let {
+            if (it !is AssertionError) it.printStackTrace()
```

Comment:
@artem-zinnatullin currently it crashes and says "Empty Test Suite" so there was no way to know that an exception was thrown or what happened. https://github.com/raniejade/spek-idea-plugin/issues/29